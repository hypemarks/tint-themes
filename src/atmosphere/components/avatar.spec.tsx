import React from 'react';
import { render } from '@testing-library/react';
import Avatar from './avatar';

jest.mock('./popup-circle', () => ({ LinkType: jest.fn() }));

describe('Test Avatar Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <svg>
        <Avatar
          type='twitter'
          xCenterPosition={0}
          yCenterPosition={0}
          avatarCircleShift={{ x: 23, y: 23 }}
          iconCircleShift={{ x: 18, y: 18 }}
          iconShift={{ x: 12, y: 12 }}
        />
      </svg>
    );
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
