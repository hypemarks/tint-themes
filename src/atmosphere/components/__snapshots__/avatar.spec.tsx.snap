// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Avatar Component should render snapshot 1`] = `
<DocumentFragment>
  <svg>
    <circle
      fill="#000"
      radius="12"
      stroke="white"
      x="18"
      y="18"
    />
    <text
      align="center"
      fill="white"
      font-family="FontAwesome"
      font-size="12"
      text="ï‚™"
      width="12"
      x="12"
      y="12"
    />
  </svg>
</DocumentFragment>
`;
